client/signup/steps/site-style/style.scss

```diff
@@ -8,7 +8,7 @@
 	align-items: stretch;
 	height: 100%;
 	background-color: var( --color-neutral-0 );
-	border-color: $gray-lighten-30;
+	border-color: var( --color-neutral-0 );
```

Comment:
What about using one of the `--color-border-*` colors?

Reply:
probably a good idea!

Reply:
hrm. weirdly that would be `--color-border-shadow` to keep the same color. We also already have quite a few borders using `--color-neutral-*` which should probably consolidate... 

Maybe a different PR?